scripts/dockerbuild.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
Added a dockerbuild.sh file , future goal is to eliminate the step of building docker images during the build process,rather consume from docker registry. @gkhanna79 has suggested opening an issue related to this.